testing/kuttl/e2e/major-upgrade-missing-image/10--cluster.yaml

```diff
@@ -0,0 +1,23 @@
+---
+# Create the cluster we will do an actual upgrade on, but set the postgres version
+# to '10' to force a missing image scenario
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: major-upgrade-empty-image
+spec:
+  # postgres version that is no longer available
+  postgresVersion: 10
+  patroni:
+    dynamicConfiguration:
+      postgresql:
+        parameters:
+          shared_preload_libraries: pgaudit, set_user, pg_stat_statements, pgnodemx, pg_cron
```

Comment:
why do we care about these shared libraries?

Reply:
This test was designed as a supplement to the `major-upgrade` test, so I tried to keep things similar where possible. I'm not sure the libraries are strictly required here, but there shouldn't be any harm in leaving them configured.

Reply:
Looking back a little - there were issues where an upgrade would fail due to extensions not being installed in the upgrade container. The upgrade test was updated to include the libraries and test this case (sc-16369).